src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -20,10 +20,10 @@ const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage })
     <PageBlock
       title={'Work Package Details'}
       headerRight={
-        <>
+        <Box display="flex" justifyContent="flex-end">
```

Comment:
last super small thing - because headerRight is formatter to already format them to the right, you can delete display="flex" justifyContent="flex-end"

Reply:
got it!